tests/dag_processing/test_manager.py

```diff
@@ -111,6 +113,7 @@ def waitable_handle(self):
 
 class TestDagFileProcessorManager:
     def setup_method(self):
+        dictConfig(DEFAULT_LOGGING_CONFIG)
```

Comment:
Should we call `logging_config.configure_logging()` instead?

Reply:
I copied it from TestStandardTaskRunner - there it is much more obvious why I used it this way. I have no super strong opinion, though I'd say in tests stating expllicitly that "this test requires default config" is a bit more resilient - for example if another test generates local_config and does not remove it - using DEFULT_* will still work.

Reply:
That's fair. At the end of the day, like normal, the only thing that is consistent is that things are inconsistent 😀.

Reply:
Indeed. This PR is a clear manifestation of it :)